README.rst

```diff
@@ -70,7 +71,7 @@ Mac/Linux
     pip install virtualenv
     virtualenv <your-env>
     source <your-env>/bin/activate
-    <your-env>/bin/pip install google-cloud-bigquery
+    <your-env>/bin/pip install google-cloud-bigquery-magics
```

Comment:
The package name will just be `bigquery-magics` as this isn't a client library corresponding with any backend REST/gRPC API.

```suggestion
    <your-env>/bin/pip install bigquery-magics
```

Reply:
Done.